ddtrace/contrib/flask/patch.py

```diff
@@ -537,7 +542,9 @@ def _wrap(code_or_exception, f):
 
 def _block_request_callable(call):
     request = flask.request
-    core.set_item(HTTP_REQUEST_BLOCKED, True)
+    import ddtrace.appsec._constants as asm_constants
```

Comment:
Including an `appsec` import in the `flask` directory confuses the separation of concerns between the Appsec Product and the Flask Contrib. I recommend refactoring such that no appsec imports are necessary in this file.

Reply:
But this function is only called and used for ASM and defining ASM constants outside of ASM is also confusing for the separation of products.

Reply:
If `_block_request_callable` is defined in appsec directory, it would still need to be imported in the next function `request_tracer`. This is the point where we plug in ASM, this is even better to be explicit.